internal/projection/process/event_enricher_test.go

```diff
@@ -299,11 +299,32 @@ var _ = Describe("Process message", func() {
 			assertHostsSummary(enrichedEvent)
 		})
 	})
+	When("Enrich a cluster event with changing version metadata", func() {
```

Comment:
Can we also add versions check in the current metadata format test? This way we make sure we will be able to process both messages in the same way

Reply:
Sorry, I think we do not have such test atm :sweat: 
It would be good to test both formats: `{"versions":{"release_tag":"foobar", "versions":{...}}}` and `{"versions":{...}}`